### ISSUE TITLE
Calibration after heating

### ISSUE THREAD

**Issue by marcopiraccini** [Apsu/cura-i3-mk2#1]
Hi, great job!
Started playing with it (I have big issues with slic3r). A couple of questions:
1) I noticed that first it check the alignement and then heats the noozle and the bed, but it's probably safer to switch the two operations
2) Material configuration is not included (I probably have to set it manually, correct?)

**Comment by marcopiraccini** [comment]
Ok, I just read "I chose to perform the Mesh Bed Leveling routine prior to heating the extruder and bed, in hopes of less oozing filament being drug around the bed by leveling after heating completes." Not sure I agree...

**Comment by Apsu** [comment]
@marcopiraccini Can you elaborate on what you disagree with exactly? By waiting for the nozzle to heat up before performing the MBL, filament can and does ooze out of the nozzle, which it then drags around the bed during the leveling. This can break off and deposit strings and globs on the bed prior to the initial wipe line.

If your concern is that the bed may change shape when hot vs cold, I don't believe that's a concern with the Mk42 bed and Mk2 bed-mounting design. The aluminum bed-mounting plate is very rigid and attaches at 6 points, one on each of the 4 corners and 2 in the middle. This is a very stable bed platform, thus I feel it makes a lot more sense to perform the MBL at the time of least risk -- when the hotend is cool and not potentially dropping filament :)

As for parking the head where it lands at the end of the MBL (Z0.15) in order to complete the heating, it's true this could potentially cause problems at that spot over time, but it's outside the printable area and not part of the nozzle-wiping strip anyway. It's also possible to have it park up a little higher then lower it prior to wiping.

**Comment by marcopiraccini** [comment]
Yes, my concern is about heating the noozle so close to the bed. Even if outside the print area, why risk?  
Maybe we could:
- Perform MBL
- Home
- Raise the head (20) 
- Heat all
- Home again
- Start

?
